### PR TITLE
Grey Buildings library: improved current layouts, added some new ones, added source files

### PR DESCRIPTION
**Pull request by Peterdevos**
Fixed origin locations on a lot of the current GB layouts. Applied transforms. Fixed position offsets when found.
Added crate, lamp, and shelf layouts. Added source files (Blender, Photoshop).

**Preview here:** https://pr-418.lba2remake.net

### CODE REVIEW

**Review by xesf — APPROVED** [review]

**Review by SendellX — COMMENTED** [review]
Well, it seems like I can't make comments in each single file, so I suppose I will be making all of them here, I hope it won't be too confusing...

The most important thing I've noticed is that several layouts are bigger than their original layout size. Staying inside the original's size is important for the physics and for preventing layout overlapping. For example:

jars01

Here you can see that the biggest jar goes out of the layout limits (the yellow line). Usually in this type of layout it doesn't matter much as it's not a place where we're supposed to place Twinsen on, but you have to be careful because if you don't have at least one brick of space between this layout and the next one, this happens:

jars02

The shelf's size is correct, so we'd need to change the jars. And while we're at it... As you have a bit of space on the sides, maybe you can separate them a little bit so they won't overlap? It's not very noticeable, but some parts of the jars go through each other.

Another thing we have to be careful with is when layouts have to match with walls, for example:

barrel02

As you can see in this one, there's a gap between the wall and the start of the layout. The handle gets a bit out of the size too, so I'd make it a bit smaller, but that one is probably not too important.

ladder01

With ladders it's important that the corner matches perfectly, otherwise this happens:

baggage01

Also, we want **tables** to match the surface of the layout size, otherwise things on top of them will be floating.

table01
table02
table03

The surface of those tables must reach the **yellow top line** if we want everything to align properly.

Another one that I believe is most important is the crate, as we can put Twinsen on top of it and there are things piled on them sometimes:

crate01

In this case it's bigger than it should.

Other examples of layouts not matching their size but that are probably not that important:

barrel01
chair01
juke02
magic01
(And same for the life points one)
plant01
For this flower I wouldn't shrink its height as we're not supposed to let Twinsen get on top of them, but I'd definitely make the leaves smaller so they won't go through walls or characters.
vase02

Other issues I've noticed...

Regarding colors:

- The barrels appear magenta and I don't get why if they're not supposed to have a texture 🤔
- We're missing the textures for the jukebox and wall decos in their .png format so Blender can read them.
- The railings don't seem to have a consistent color:
railings01
I've checked them and the material color is indeed the same. The theory I have is that the color you picked is in between of two colors of the original palette, so it picks one of the closest but it's not consistent on which one it picks. I used the eye dropper tool and it says that the original has the **8,11** color, so I'd suggest that you append that material from the master scene and replace the "Railing" material you have for that 8,11 color. If this doesn't help just submit it as it is and I'll investigate why further.

And now issues regarding modeling:

- It's a very subtle thing, but the jukebox has a shadow due to different mesh density. It's not important at all, but I just can't stop seeing it... 😅
juke01
As the button and the slot are separated islands (good one 👍), you should be able to match one side to the other very easily. I'd add the same geometry you have in the slot side to the other one.

- Tiny glitch in the flower that makes it look like a cute face 💖
plant02
Again, not important at all, probably I'll be the only one noticing these things 😂

- I really love how you resolved the vase, but maybe I'd do something for avoiding the big difference between the meshes' angles. Maybe something like this...
vase01

- Why is the window missing the black cover in the right side? 🤔
window01

- I don't know what you think, but I'd make these hard edges in these bottle's content for making it look like liquid, just like you did with the round ones. I think it looks less confusing. 
bottles01
About those hard edges in the inner side of the glass, right now they don't have impact because we have the transparency loading order issue, but I think it'll look weird once we fix it... I suppose we'll worry about it then.

- I found this liiiitle tiny visual glitch 🙈 It's happening because of those hard edges in those indentations.
arch02

- I want to ask you about this. I really love this assimetry in the crate's sides, but why are the faces going through each other like that in these joints? Is there some artistic purpose behind? It looks very weird to me. I know nobody will notice this probably but I'm very curious why you did it like this.
crate02

I think that's all. Despite all I wrote I think you're doing an awesome job, and of course if you don't agree to what I'm saying here I'd like to know so we can reach an agreement. You don't have to worry about the minor stuff, but I believe that matching the sizes at least is important...

File: assetsSource/Layouts/GB/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+GB_railings.blend=@GB_railings.blend,0
+GB_jukebox.blend=@GB_jukebox.blend,0
+GB_cash_register.blend=@GB_cash_register.blend,0
+GB_wall_lamp.blend=@GB_wall_lamp.blend,0
+GB_bottles_glasses.blend=@GB_bottles_glasses.blend,0
+GB_health_magic.blend=@GB_health_magic.blend,0
+GB_plant.blend=@GB_plant.blend,0
+GB_stairs.blend=@GB_stairs.blend,0
+GB_window.blend=@GB_window.blend,0
+GB_chairs_tables.blend=@GB_chairs_tables.blend,0
```
[the remark this answers is not in the document]
Comment (xesf) [reply assetsSource/Layouts/GB/desktop.ini:18]: Nop, this file is Windows specific and should not be tracked on git.

[the remark this answers is not in the document]
Comment (Peterdevos) [reply assetsSource/Layouts/GB/desktop.ini:18]: Okay, I'll delete it with my next commit!

[the remark this answers is not in the document]
Comment (agrande) [reply assetsSource/Layouts/GB/desktop.ini:18]: You may want to add `desktop.ini` to the gitignore list, so it doesn't get in the way, here's how you can do it with GitHub Desktop: https://caltechlibrary.github.io/git-desktop/05-ignore/#:~:text=To%20ignore%20files%20in%20your,to%20ignore%20in%20your%20repository.

**Pull request merged**